infrastructure/smart-contracts/contracts/upgrades/referee/Referee9.sol

```diff
@@ -395,7 +395,7 @@ contract Referee9 is Initializable, AccessControlEnumerableUpgradeable {
         uint256 maxSupply = Xai(xaiAddress).MAX_SUPPLY();
 
         // Get the timestamp of the start of the current challenge
-        uint256 startTs = challenges[challengeCounter - 1].assertionTimestamp;
```

Comment:
Are these the correct timestamp variables? This seems like we're going from the assertion timestamp to the challenge timestamp. If that's intended then ignore this comment.